common/autoconf/flags.m4

```diff
@@ -166,6 +166,26 @@ AC_DEFUN_ONCE([FLAGS_SETUP_COMPILER_FLAGS_FOR_LIBS],
     C_FLAG_REORDER=''
     CXX_FLAG_REORDER=''
 
+    if test "x$OPENJDK_TARGET_OS" = xmacosx; then
```

Comment:
This file has an IBM copyright notice: the end date should be updated.

Reply:
even though this change wasn't an IBM change?

Reply:
it's ok, it now has an IBM change :-)

Reply:
It had an IBM change before, hence the presence of the copyright notice.